contrib/cirrus/lib.sh

```diff
@@ -194,6 +195,37 @@ setup_rootless() {
            -o CheckHostIP=no true
 }
 
+# This should be the first thing run from any test. It's only really
+# applicable for rootless, because we've ssh'ed in and don't have a
+# working set of environment variables.
+load_cirrus_task_environment() {
+    envfile="$HOME/cirrus-task-environment"
+
+    # The usual case: we already have the environment we need
+    if [ ! -e $envfile ]; then
```

Comment:
Would this be an improvement:

`if [[ ! -e $envfile ]] && ((SETUP_ENVIRONMENT)); then`